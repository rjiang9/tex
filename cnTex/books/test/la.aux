\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\@writefile{toc}{\contentsline {chapter}{索引}{4}{chapter*.1}}
\@writefile{toc}{\contentsline {section}{章节描述}{4}{section*.2}}
\@writefile{lof}{\contentsline {figure}{插图描述}{4}{section*.2}}
\@writefile{lot}{\contentsline {table}{表格描述}{4}{section*.2}}
\@writefile{toc}{\contentsline {chapter}{文\hskip 1em\relax 献}{4}{section*.2}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces 浮动图片自动调整位置}}{5}{figure.0.1}}
\newlabel{fig:float}{{1}{5}{浮动图片自动调整位置}{figure.0.1}{}}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces 浮动表格例子}}{5}{table.0.1}}
\newlabel{tab:float}{{1}{5}{浮动表格例子}{table.0.1}{}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces 浮动表格例子}}{5}{table.0.2}}
\newlabel{tab:float}{{2}{5}{浮动表格例子}{table.0.2}{}}
\@writefile{lot}{\contentsline {table}{\numberline {3}{\ignorespaces 浮动表格例子}}{5}{table.0.3}}
\newlabel{tab:float}{{3}{5}{浮动表格例子}{table.0.3}{}}
\citation{DK1}
\citation{DK2}
\@writefile{toc}{\contentsline {chapter}{\numberline {第一章\hspace  {.3em}}章标题}{7}{chapter.1}}
\@writefile{lof}{\addvspace {10.0pt}}
\@writefile{lot}{\addvspace {10.0pt}}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}节标题}{7}{section.1.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.1}小节标题}{7}{subsection.1.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{子节标题}{7}{subsubsection*.3}}
\@writefile{toc}{\contentsline {paragraph}{段标题}{7}{paragraph*.4}}
\@writefile{toc}{\contentsline {subparagraph}{小段标题}{7}{subparagraph*.5}}
\bibcite{DK1}{mag}
\bibcite{DK2}{ray}
\bibcite{DK3}{Knuth3}
