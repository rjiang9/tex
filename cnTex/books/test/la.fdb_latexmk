# Fdb version 3
["pdflatex"] 1519943520 "la.tex" "la.pdf" "la" 1519943520
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/adobe/zapfding/pzdr.tfm" 1136797454 1528 7e12f4c0fcf5a072cebdcfbfc610f54c ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmmi12.tfm" 1136765053 1524 4414a8315f39513458b80dfc63bff03a ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmmi6.tfm" 1136765053 1512 f21f83efb36853c0b70002322c1ab3ad ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmmi8.tfm" 1136765053 1520 eccf95517727cb11801f4f1aee3a21b4 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmr10.tfm" 1136765053 1296 45809c5a464d5f32c8f98ba97c1bb47f ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmr12.tfm" 1136765053 1288 655e228510b4c2a1abe905c368440826 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmr6.tfm" 1136765053 1300 b62933e007d01cfd073f79b963c01526 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmr8.tfm" 1136765053 1292 21c1c5bfeaebccffdb478fd231a0997d ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmsy10.tfm" 1136765053 1124 6c73e740cf17375f03eec0ee63599741 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmsy6.tfm" 1136765053 1116 933a60c408fc0a863a92debe84b2d294 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/cm/cmsy8.tfm" 1136765053 1120 8b7d695260f3cff42e636090a8002094 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/latex-fonts/lasy6.tfm" 1136765053 520 4889cce2180234b97cad636b6039c722 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xyatip10.tfm" 1381018713 608 50246cc71b0635b0ba0a5c10a0bf4257 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xybsql10.tfm" 1381018713 608 4db60f15ea23b4ec2d796c6d568a63fa ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xybtip10.tfm" 1381018713 608 50246cc71b0635b0ba0a5c10a0bf4257 ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xycirc10.tfm" 1381018713 844 3393210079fb4ed9347e214b3bfd7c1a ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xycmat10.tfm" 1381018713 608 f124f78ed50a1817738d2adb190cf2bd ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xycmbt10.tfm" 1381018713 608 f124f78ed50a1817738d2adb190cf2bd ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xydash10.tfm" 1381018713 984 5c01c46b93e3ba8369f3f8edc6e62aef ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xyluat10.tfm" 1381018713 608 a3a3bc08980c5126ff2a7a68fb5a64ff ""
  "C:/Program Files/MiKTeX 2.9/fonts/tfm/public/xypic/xylubt10.tfm" 1381018713 608 a3a3bc08980c5126ff2a7a68fb5a64ff ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/ifxetex/ifxetex.sty" 1284257189 1458 43ab4710dc82f3edeabecd0d099626b2 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/oberdiek/gettitlestring.sty" 1489930824 8237 3b62ef1f7e2c23a328c814b3893bc11f ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/oberdiek/hobsub-generic.sty" 1489930824 185082 1fb09d7d24834377f95006300bc91fd2 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/oberdiek/hobsub-hyperref.sty" 1489930824 70864 bcd5b216757bd619ae692a151d90085d ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/oberdiek/stringenc.sty" 1489930828 21411 4c6135fd98dd4f5499becbfc9a323deb ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/unicode-data/CaseFolding.txt" 1498574222 78161 7893b6e005c5a521319a0d12062ae122 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/unicode-data/SpecialCasing.txt" 1498574222 16830 e49d6f1fe536f43762ba1ecfe9e53e12 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/unicode-data/UnicodeData.txt" 1498574222 1735010 2a52f30695dcc821f0f224650552beaf ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xy.sty" 1381018713 4692 1e1bcf75c622af1eefd9169948208302 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xy.tex" 1381018713 115380 413d5f789929a45aab7d12ce0d0aee7d ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xyall.tex" 1381018713 1449 24340b6befc66d28ee1ebb657efb5892 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xyarrow.tex" 1381018713 22657 990ce136a3cc15728ba417a2e78b25c8 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xycmtip.tex" 1381018713 1374 43fb8dc80dd748631d78096701166d76 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xycolor.tex" 1381018713 4586 edd672434f45626662368282c0322160 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xycurve.tex" 1381018713 109670 d412ee1ff259daefee5e927172e2f9a8 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xyframe.tex" 1381018713 24249 186931a828664624939ab0b347e3952c ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xygraph.tex" 1381018713 9619 b7e4d9a6936ba2ad6119a280abde9641 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xyidioms.tex" 1381018713 2907 1ee562fde0b53c9cd16f7a604f33fdf0 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xyline.tex" 1381018713 10928 c3a572983ccc9fc596b4e9ce454d5652 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xymatrix.tex" 1381018713 22583 25b1e7edeee41f181ee9733429da4a9c ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xyrecat.tex" 1381018713 1391 c8763fc8e281cb6ecf697988b6608e4a ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xyrotate.tex" 1381018713 7008 cb768d8d63a12d35607cbb3c4e7ba163 ""
  "C:/Program Files/MiKTeX 2.9/tex/generic/xypic/xytips.tex" 1381018713 3689 0d51788a4141bc66ab896f7ac63495fd ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/00miktex/hyperref.cfg" 1136242423 235 6031e5765137be07eed51a510b2b8fb7 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/amsmath/amscd.sty" 1504971625 5455 f89f20c42255849b5c46f9d3a0d4d1f5 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/amsmath/amsgen.sty" 1504971626 4296 c115536cf8d4ff25aa8c1c9bc4ecb79a ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/base/bk12.clo" 1492211338 8139 b9d9424b409fe55c665eac87eba27e54 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/base/book.cls" 1492211338 22945 7df8250bf76418cf651c357d3631fbc3 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/base/fix-cm.sty" 1492211346 14742 ad7ac4c2db1d39579bbccd175cdf5ca9 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/base/fontenc.sty" 1492211346 4571 13977df0eda144b93597fc709035ad1f ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/base/ts1enc.def" 1492211342 7767 aa88823823f5e767d79ea1166ab1ae74 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/base/tuenc.def" 1492211342 27166 081ef1345bf363faf81b61c2450b871f ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/cjk/CJKfntef.sty" 1429336288 12851 a92a26031294f6fa88114f524b3b794a ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/config/ctex-name-utf8.cfg" 1517163322 1873 1dc24aea7b580193e7efd56c6ca38a49 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/config/ctex.cfg" 1517163322 1334 b488a91b76f715de9957b1ef55206d0f ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/config/ctexopts.cfg" 1517163322 1405 10c85e122705aee852db17b916379b15 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/ctexbook.cls" 1517163320 53300 d3368f9bf510d62a422938d48457c06c ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/ctexhook.sty" 1517163320 2768 667d97686182304498bce1043de8c995 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/ctexpatch.sty" 1517163320 8526 9fdb9a65852b503e6d827576107094ee ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/engine/ctex-engine-xetex.def" 1517163322 5686 ee6d79f2da2a9d0a6a9ae5552a513697 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/fontset/ctex-fontset-windows.def" 1517163322 1538 7afec8bdf342748bb8d7f4d39b3e44d9 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/fontset/ctex-fontset-windowsnew.def" 1517163322 5071 b4eba1fec152f7a598d1d8044ef48d68 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ctex/scheme/ctex-scheme-chinese-book.def" 1517163322 5406 829e6eded1c05d0a842a201d2808c1d7 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/environ/environ.sty" 1399188561 4378 f429f0da968c278653359293040a8f52 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/fontspec/fontspec-xetex.sty" 1509918720 151582 64a2902c50cf545a531f4a5f48452140 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/fontspec/fontspec.cfg" 1509918720 550 e2d151424a668bd78dad72056f87216f ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/fontspec/fontspec.sty" 1509918720 1567 730e34e802c841e3801c5f4d9f6090df ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/geometry/geometry.sty" 1284467836 41632 e003406220954b0716679d7928aedd8a ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics-cfg/color.cfg" 1465890692 1213 620bba36b25224fa9b7e1ccb4ecb76fd ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics-cfg/graphics.cfg" 1465890692 1224 978390e9c2234eab29404bc21b268d1e ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics-def/xetex.def" 1498375874 12707 672c620846df0fecf0a7a99fed83566d ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics/color.sty" 1498479285 7229 4ca2945c73eec4e726c0da188fbbf50f ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics/graphics.sty" 1498479285 15796 7d676729b1bedd3e7f3c6717affb366c ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics/graphicx.sty" 1498479285 9342 649f2ccf62888e3d8c3e57256b70b8e1 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics/keyval.sty" 1498479285 2681 d18d5e19aa8239cf867fa670c556d2e9 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/graphics/trig.sty" 1498479285 4097 0a268fbfda01e381fa95821ab13b6aee ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/hyperref/hxetex.def" 1489913924 45109 0d162ebe2b85589994e9a71add12b9b3 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/hyperref/hyperref.sty" 1489913924 233808 b63d91422c362e723c6e8b1a2fffcba5 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/hyperref/nameref.sty" 1489913924 12949 81e4e808884a8f0e276b69410e234656 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/hyperref/pd1enc.def" 1489913924 14098 7631f11156e5f9cd76010dbd230aa268 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/hyperref/puenc.def" 1489913924 122411 7c1f7e015c7fc4bfb7dc052d55755ce3 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/l3kernel/expl3-code.tex" 1513456130 851718 4c5d975e420639ac461f1256b65c9b0c ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/l3kernel/expl3.sty" 1513456130 9854 7c3f310f5809fb117c635071bb19c106 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/l3kernel/l3xdvipdfmx.def" 1513456130 14168 629291596470f459a7722531a5ef2be1 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/l3packages/l3keys2e/l3keys2e.sty" 1513457676 4568 c0814c5057d6a90af9ab1e2fa255dac4 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/l3packages/xparse/xparse.sty" 1513457920 76828 7c9a32558c0fffc46742c4b8e3d51903 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/l3packages/xtemplate/xtemplate.sty" 1513457994 48896 9e9664de372a60df1851447a0b4a0a87 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ms/everysel.sty" 1320143990 4752 936bfb87af7b91b6496a2317f80db16b ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/oberdiek/auxhook.sty" 1489930830 3834 4363110eb0ef1eb2b71c8fcbcdb6c357 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/oberdiek/kvoptions.sty" 1489930832 22417 1d9df1eb66848aa31b18a593099cf45c ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/oberdiek/rerunfilecheck.sty" 1489930834 9581 023642318cef9f4677efe364de1e2a27 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/tipa/t3cmr.fd" 1080618240 1257 fd5b1c2f0106f9cf12cdb2ff961987c5 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/tipa/t3enc.def" 1080618240 26483 c461a68c34040cb0d96c43e86c012a6d ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/trimspaces/trimspaces.sty" 1253165583 1380 971a51b00a14503ddf754cab24c3f209 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/ulem/ulem.sty" 1338506272 23756 854c01b779030ff5b2aad88ba7a119f2 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/url/url.sty" 1388490452 12796 8edb7d69a20b857904dd0ea757c14ec9 ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/zhnumber/zhnumber-utf8.cfg" 1517163752 5585 3b6ee8d33ca38c2ca7167f650fed047d ""
  "C:/Program Files/MiKTeX 2.9/tex/latex/zhnumber/zhnumber.sty" 1517163752 32080 9e8968e54bbbada46eb61edc967f578c ""
  "C:/Program Files/MiKTeX 2.9/tex/xelatex/xecjk/xeCJK.cfg" 1517163510 408 dbad970bb8db25d4bd4bba27f088d79c ""
  "C:/Program Files/MiKTeX 2.9/tex/xelatex/xecjk/xeCJK.sty" 1517163510 177915 0547fe954bc0f2a524fa1f007ab3d89e ""
  "C:/Program Files/MiKTeX 2.9/tex/xelatex/xecjk/xeCJKfntef.sty" 1517163510 40256 a415c92d56c87eab64f81b4c0544f79c ""
  "C:/Program Files/MiKTeX 2.9/tex/xelatex/xecjk/xunicode-addon.sty" 1517163510 23594 6d554f32f08c1076f2d9f79a989f1168 ""
  "C:/Program Files/MiKTeX 2.9/tex/xelatex/xecjk/xunicode-extra.def" 1517163510 61599 23d20cd8ab176fa4f4bbd84e9841123b ""
  "C:/Program Files/MiKTeX 2.9/tex/xelatex/xetexconfig/geometry.cfg" 1169251881 322 f4a6234bc28bf70659404393c15d3854 ""
  "C:/Program Files/MiKTeX 2.9/tex/xelatex/xunicode/xunicode.sty" 1315547446 205722 8027ed7ab2230086d647802c25f3ee42 ""
  "C:/Users/rjiang/AppData/Local/MiKTeX/2.9/miktex/data/le/xetex/xelatex.fmt" 1518018231 15518486 650ea59eb8998df52d9ab961d845399a ""
  "C:/Windows/Fonts/msyh.ttc" 1464365418 21543568 ff132bc70e4c9b54fad28218a4b9b0d4 ""
  "la.aux" 1519943509 2416 5303acb99f3c279dd7ff2ed13bee6115 ""
  "la.out" 1519943509 458 991e3843504db72986355ee4d3780156 ""
  "la.tex" 1519943519 6711 d38c843f9aae53d70976a168ec6f74d3 ""
  "la.toc" 1519943509 590 36e0c29145dafcb74c9efef99114c03a ""
  "picture/sea.jpg" 1467289856 32814 c3bc53d4607cabab7cd7db8b6bd370cf ""
  (generated)
  "la.out"
  "la.toc"
  "la.idx"
  "la.lof"
  "la.pdf"
  "la.lot"
  "la.aux"
  "la.log"
