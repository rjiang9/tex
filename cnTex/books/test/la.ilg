This is makeindex, version 2.15 [MiKTeX 2.9.6500 64-bit] (kpathsea + Thai support).
Scanning input file la.idx...done (0 entries accepted, 0 rejected).
Nothing written in la.ind.
Transcript written in la.ilg.
