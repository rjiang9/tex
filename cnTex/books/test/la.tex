
% la.tex
\documentclass[UTF8,a5paper,12pt,portrait,openary,final]{ctexbook}
\usepackage[colorlinks=true,
            linkcolor=black
            ]{hyperref}                               %目錄跳轉
% documentclass not change page size, this one does
\usepackage[b5paper,text={125mm,195mm},centering]{geometry}
%\usepakage{graphics}
%\usepakage{graphicx}

%\usepackage{makeidx}     % -- for 檢索 --
%\makeindex %  生成检索
%\setlength{\baselineskip}{24pt}  % 修改行距
%在编译时，先用 xelatex/pdflatex 编译，接着用 makeindex 编译，最后再
%用 xelatex/pdflatex 编译，才能得到带全文索引的文档。


\usepackage{amscd}      % 画交换图
\usepackage[all]{xy}     % 画交换图的另一个包




\begin{document}


\title{Notes On LaTeX Typesetting}
\author{Some One}
\date{November 13, 2011}
\maketitle


\ctexset{
abstractname = {本文概要},
bibname = {文\quad 献}
}

\tableofcontents    % 目录
%\listoffigures      % 插图目录
%\listoftables       % 表格目录


\clearpage
\addcontentsline{toc}{chapter}{\indexname}
\addcontentsline{toc}{section}{章节描述}
\addcontentsline{lof}{figure}{插图描述}
\addcontentsline{lot}{table}{表格描述}
\addcontentsline{toc}{chapter}{\bibname}

\printindex
\newpage


\xymatrix{
M \ar[rr]^{f}\ar[dr]_{h} &   & N \ar@{-->}[dl]^{g}\\
& P &
}

\xymatrix{
M &   & N \\
& P &
}

\[
\[
\begin{CD}
DT @>f_1>> BJ \\
@Vf_2VV @AAg_1A  \\
HZ @<<g_2< HD
\end{CD}
\]
\]

\[
\[
\begin{CD}
A @>f_1>> B \\
@Vf_2VV @AAg_1A  \\
C @<<g_2< D
\end{CD}
\]
\]









\setlength{\baselineskip}{12pt plus 2pt minus 1pt}

在在T E X中，默认的行距通常是字体大小的1.2倍。如果字体大小为10pt，默认行距 baselineskip将为12pt
。若要调整行距，可以直接修改 baselineskip
的值。例如下面命令将调整行距为双倍行距：

\setlength{\baselineskip}{4pt}
其中，在L A T E X中\setlength命令用于修改各种长度的值。T E X中，默认的行距通常是字体大小的1.2倍。如果字体大小为10pt，默认行距baselineskip将为12pt。若要调整行距，可以直接修改
baselineskip的值。例如下面命令将调整行距为双倍行距
命令用于修改各种长度的值。
命令用于修改各种长度的值。命令用于修改各种长度的值。命令用于修改各种长度的值。

{\linespread{4}\selectfont 命令用于修改各种长度的值。
其中，在L A T E X中setlength
命令用于修改各种长度的值。
}

\linespread{1}\selectfont

\begin{figure}[htbp!]
\centering
\includegraphics[scale=0.3]{picture/sea.jpg}
\caption{浮动图片自动调整位置}
\label{fig:float}
\end{figure}


\\


\renewcommand\arraystretch{1.5}     % 调整表格行距
\begin{table}[htbp!]
\centering
\begin{tabular}{|l|c|r|l|}
\hline
第一行&第一行&第一行&第一行\\
\hline
第二行&第二行&第二行&第二行\\
\hline
第一行&第一行&第一行&第一行\\
\hline
第二行&第二行&第二行&第二行\\
\hline
\end{tabular}
\caption{浮动表格例子}
\label{tab:float}
\end{table}

\\ 
\begin{table}[htbp!]
\centering
\begin{tabular}{|c|c|c|}
\hline
第一行&第一行&第一行\\
\hline
\multicolumn{2}{|c|}{跨列} &第二行\\
\hline
第三行&第三行&第三行\\
\hline
第四行 & \multicolumn{2}{|c|}{跨列}\\
\hline
\end{tabular}
\caption{浮动表格例子}
\label{tab:float}
\end{table}



\\
\begin{table}[htbp!]
\centering
\begin{tabular}{|l|c|r|}
\hline
左列&中列&右列\\
\hline
第二行&第二行&第二行\\
\hline
左三&中三&右三\\
\hline
\end{tabular}
\caption{浮动表格例子}
\label{tab:float}
\end{table}





正常在黄昏融化了世界的色彩以前正常在黄
\raisebox{-2mm}{\includegraphics[scale=0.03]{picture/sea.jpg}}
昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前
正常中国人民解放军\rotatebox{25}{逆时针}正常在黄昏融化了世界的色彩以前
正常在黄昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前
正常中国人民解放军\rotatebox{25}{逆时针}正常在黄昏融化了世界的色彩以前
正常在黄昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前正常在黄昏融化了世界的色彩以前
\\
正常\rotatebox{-30}{顺时针}正常
\\
正常\scalebox{-1}[1]{翻转}正常
\\
正常\reflectbox{翻转}正常
\\

正常\scalebox{-1.6}[0.7]{矮胖}正常
\\
正常\scalebox{0.7}[-1.6]{瘦高}正常
\\



正常\scalebox{1.6}[0.7]{矮胖}正常
\\
正常\scalebox{0.7}[1.6]{瘦高}正常
\\
正常\scalebox{1.6}{胖高}正常

\\

正常\resizebox{30pt}{5pt}{矮胖}正常
\\正常\resizebox{11pt}{13pt}{瘦高}正常
\\正常\resizebox{30pt}{!}{胖高}正常




\\
中华\\
人民{\huge 共和国}\\
解放军\smash{军部} 关门

\\







\chapter{章标题}
这一章我们介绍这些内容。
\section{节标题}
这一节我们介绍这些内容。
\subsection{小节标题}
这一小节我们介绍这些内容。
\subsubsection{子节标题}
这一子节我们介绍这些内容。
\paragraph{段标题}
这一段我们介绍这些内容。

D. Knuth wrote some books, e.g. \cite{DK1, DK2}.
\subparagraph{小段标题}
这一小段我们介绍这些内容。







\\
正文正文正文\\
正文\raisebox{6pt}[30pt][3pt]{\fbox{上升}}正文


\\


他的师父是\raisebox{10pt}{\huge 上}真\raisebox{3pt}{\fbox{下}}华
\\
正文\raisebox{-6pt}{\fbox{下降}}正 {\tiny 文}

\\



正文正文
\\
正文\hphantom{水平幻影}正文
\\
正文\vphantom{\Huge竖直幻影}正文



\fbox{a} \fbox{A} \quad
\fbox{a\rule{0pt}{7pt}}
\fbox{A\rule{0pt}{7pt}}
默认\rule{3pt}{14pt}默认
上升\rule[5pt]{3pt}{14pt}上升
下降\rule[-5pt]{3pt}{14pt}下降
\\


正文\mbox{盒子}正文\fbox{盒子}正文

正文
\makebox[10em][r]{靠右对齐}正文\\
正文\framebox[10em][r]{靠右对齐}正文

Left\begin{minipage}{6em}
One\\ Two\\ Three
\end{minipage}Right

\begin{abstract}
some abstract
\end{abstract}


Left\parbox{6em}{One\\ Two\\ Three}Right

\\
Left\fbox{\begin{minipage}{6em}
One\\ Two\\ Three
\end{minipage}}Right


\newtheorem{aaa}{Theorem}
\newtheorem{bbb}{Corollary}

\begin{aaa}
This is a theorem.
\end{aaa}
\begin{bbb}
This is a corollary.
\end{bbb}

\begin{thebibliography}{123456}
\bibitem[mag]{DK1} D. Li, T.A.O.C.P. , Vol. 1, Addison-Wesley, 1997.
\bibitem[ray]{DK2} D. Jiang, T.A.O.C.P. , Vol. 2, Addison-Wesley, 1999.
\bibitem[Knuth3]{DK3} D. Knuth, T.A.O.C.P. , Vol. 3, Addison-Wesley, 1998.
\end{thebibliography}

\end{document}

