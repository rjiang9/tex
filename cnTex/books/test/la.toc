\contentsline {chapter}{索引}{4}{chapter*.1}
\contentsline {section}{章节描述}{4}{section*.2}
\contentsline {chapter}{文\hskip 1em\relax 献}{4}{section*.2}
\contentsline {chapter}{\numberline {第一章\hspace {.3em}}章标题}{7}{chapter.1}
\contentsline {section}{\numberline {1.1}节标题}{7}{section.1.1}
\contentsline {subsection}{\numberline {1.1.1}小节标题}{7}{subsection.1.1.1}
\contentsline {subsubsection}{子节标题}{7}{subsubsection*.3}
\contentsline {paragraph}{段标题}{7}{paragraph*.4}
\contentsline {subparagraph}{小段标题}{7}{subparagraph*.5}
