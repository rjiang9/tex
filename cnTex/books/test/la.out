\BOOKMARK [0][-]{chapter*.1}{\376\377\175\042\137\025}{}% 1
\BOOKMARK [1][-]{section*.2}{\376\377\172\340\202\202\143\317\217\360}{chapter*.1}% 2
\BOOKMARK [0][-]{section*.2}{\376\377\145\207\000\040\163\056}{}% 3
\BOOKMARK [0][-]{chapter.1}{\376\377\172\340\150\007\230\230}{}% 4
\BOOKMARK [1][-]{section.1.1}{\376\377\202\202\150\007\230\230}{chapter.1}% 5
\BOOKMARK [2][-]{subsection.1.1.1}{\376\377\134\017\202\202\150\007\230\230}{section.1.1}% 6
